### DISCUSSION TITLE
Homebrew tap publishing error

### DISCUSSION THREAD

**Discussion by siddhant-deepsource** [goreleaser/goreleaser#2387] [Q&A] [answered]
Hey all,

First of all, thanks a lot for this awesome project. Its really helpful and gets lot of tough tasks done in such easy way.

My previous concern was fixed.

Just had a new query that is it possible to push to a tap repo and create a PR instead of directly pushing to master branch? Or maybe to a different branch which we can specify in the config?

Reason: Due to their compliance, many projects dont allow direct push to protected branches (master/main)

**Comment by caarlos0** [comment, marked as answer]
There's no way currently... opening an issue for that: https://github.com/goreleaser/goreleaser/issues/2388
